src/main/java/com/zendesk/maxwell/bootstrap/MaxwellBootstrapUtility.java

```diff
@@ -161,7 +164,7 @@ private Long calculateRowCount(Connection connection, String db, String table) t
 	private long insertBootstrapStartRow(Connection connection, String db, String table, Long totalRows) throws SQLException {
 		LOGGER.info("inserting bootstrap start row");
 		String sql = "insert into `bootstrap` (database_name, table_name, total_rows) values(?, ?, ?)";
-		PreparedStatement preparedStatement = connection.prepareStatement(sql);
+		PreparedStatement preparedStatement = connection.prepareStatement(sql, Statement.RETURN_GENERATED_KEYS);
```

Comment:
yeah, I guess there was some slop in the old code that allowed it to work without